release-engineering/role-handbooks/branch-manager.md

```diff
@@ -416,6 +417,82 @@ See the [Security Release Process](https://git.k8s.io/security/security-release-
 
 [security-release-team]: https://groups.google.com/a/kubernetes.io/forum/#!forum/security-release-team
 
+#### Announcing Security Fixes
+
+The security fixes are usually announced upon publishing releases by:
+
+- including information about security fixes in the release announcement sent to kubernetes-announce and dev mailing lists
+- including information about security fixes in CHANGELOGs for affected releases
+- sending an email announcement for each security fix to the relevant mailing lists (done by the Security Response Committee)
+
+The first two are handled by Release Managers in coordination with the Security Response Committee (SRC). Before starting the release process, the Release Managers need to reach out to the SRC and ask them to provide a CVE data map that contains information about security fixes.
```

Comment:
We only require the information from the map, right? The map itself gets created by the Release Manager.

Reply:
@saschagrunert So far, the process was that we send the map template to the SRC and they create the data map based on the provided template.